test/source/tests/tests/account.ts

```diff
@@ -59,6 +59,15 @@ export const defineConsumerAcctTests = (testVariant: TestVariant, testWithNewBro
 
     ava.test.todo('settings > subscribe > expire > compose > footer > subscribe');
 
-  }
+  } else {
+    ava.test('compose > large file > gmail account (regular user)', testWithSemaphoredGlobalBrowser('compose', async (t, browser) => {
```

Comment:
Looks good, just for clarity let's call it `compose > large file > public domain account (should not prompt to upgrade)`

Reply:
Okay, I'm renaming it